application/ldap/asg.tf

```diff
@@ -75,11 +75,16 @@ resource "aws_autoscaling_group" "asg" {
     data.terraform_remote_state.vpc.vpc_private-subnet-az3,
   )}"]
   min_size             = "1"
-  max_size             = "1"
-  desired_capacity     = "1"
+  desired_capacity     = "3"
```

Comment:
Do we have support for this yet in the LDAP server configuration?

Reply:
Yep, see https://github.com/ministryofjustice/hmpps-delius-core-ldap-bootstrap/pull/5